src/main/scala/edu/ie3/simona/config/ConfigFailFast.scala

```diff
@@ -549,6 +554,56 @@ case object ConfigFailFast extends LazyLogging {
     }
   }
 
+  /** Check the validity of control scheme definitions
+    *
+    * @param control
+    *   Control scheme definitions
+    */
+  private def checkControlSchemes(control: Simona.Control): Unit = {
+    control.transformer.foreach(checkTransformerControl)
+  }
+
+  /** Check the suitability of transformer control group definition.
+    *
+    * One important check cannot be performed at this place, as input data is
+    * not available, yet: Do the measurements belong to a region, that can be
+    * influenced by the transformer?
```

Comment:
That's part of what happens in `GridAgentFailFast`, right? You could link it here with a short hint and `[[edu.ie3.simona.agent.grid.GridAgentFailFast]]`

Reply:
Yes, at least partly - is it possible to measure the influence. I've to thing about if there is a corner case where the measurement belongs to same subgrid as the transformer but can not be influenced by it for some reason.

Reply:
I'd just plainly assume that measurements can always be influenced by the tranformers in that sub grid. Even if there are corner cases where that is not true, in my opinion it's ok to ignore those here. It's also something that I think I wouldn't require to be tested in a failFast context, as this goes beyond simple plausibility checking.